pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -353,8 +354,12 @@ func (s *nsmgrSuite) Test_ConnectToDeadNSEUsecase() {
 	defer closeCancel()
 	_, _ = nsc.Close(closeCtx, conn)
 
+	nseRegistryClient := registryclient.NewNetworkServiceEndpointRegistryClient(ctx,
+		registryclient.WithClientURL(s.domain.Nodes[0].NSMgr.URL),
+		registryclient.WithDialOptions(grpc.WithTransportCredentials(insecure.NewCredentials())),
+	)
```

Comment:
Because this registry client puts path to gRPC metadata, and registry server checks that path is presented in gRPC metadata. If we used client I deleted, we would get error from registry server that it doesn't have path in gRPC metadata